firefox_ui_tests/remote/security/test_mixed_content.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestMixedContent(FirefoxTestCase):
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_url = 'https://mozqa.com/data/firefox/security/mixedcontent.html'
+
+    def tearDown(self):
+        FirefoxTestCase.tearDown(self)
+
+    def test_mixed_content(self):
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.test_url)
+        favicon = self.browser.navbar.locationbar.favicon
+
+        def get_list_style_image(favicon):
+            favicon_image = self.marionette.execute_script("""
+                    return arguments[0].ownerDocument.defaultView
+                    .getComputedStyle(arguments[0])
+                    .getPropertyValue('list-style-image')
+            """, script_args=[favicon])
```

Comment:
@Ethcelon did you see this comment? I think we had a mid-air collision, sorry about that.